EIPS/eip-5058.md

```diff
@@ -194,7 +194,7 @@ Test case written using hardhat: [here](https://github.com/radiocaca/ERC721L/blo
 
 ## Reference Implementation
 
-You can find an implementation of this standard in the assets folder: [RadioCaca](https://github.com/radiocaca/ERC721L/tree/main/contracts/EIP5058).
+You can find an implementation of this standard in the [assets](../assets/eip-5058) folder.
```

Comment:
These should probably be individually linked. Also, this is a pretty comprehensive reference implementation - they generally should be somewhat smaller than this (although, I too like making my reference implementations way too long).

Reply:
You don't have to remove it, but linking to a folder doesn't work correctly so you'll need to link to the individual files.

Reply:
Got it, I've linked to the individual  files.